store/provide.go

```diff
@@ -18,56 +18,92 @@
 package store
 
 import (
+	"regexp"
 	"time"
 
+	"github.com/xmidt-org/argus/auth"
 	"github.com/xmidt-org/themis/config"
 	"go.uber.org/fx"
 )
 
-// StoreIn is the set of dependencies for this package's components
-type StoreIn struct {
+type handlerIn struct {
 	fx.In
 
-	Unmarshaler             config.Unmarshaller
-	Store                   S
-	AccessLevelAttributeKey string `name:"access_level_attribute_key"`
+	Store  S
+	Config *transportConfig
 }
 
-// StoreOut is the set of components emitted by this package
-type StoreOut struct {
-	fx.Out
+// ProvideHandlers fetches all dependencies and builds the four main handlers for this store.
+func ProvideHandlers() fx.Option {
+	return fx.Provide(
+		newAccessLevelAttributeKeyAnnotated(),
+		newTransportConfig,
 
-	// SetItemHandler is the http.Handler to update an item in the store.
-	SetItemHandler Handler `name:"setHandler"`
+		fx.Annotated{
+			Name:   "setHandler",
+			Target: newSetItemHandler,
+		},
+		fx.Annotated{
+			Name:   "getHandler",
+			Target: newGetItemHandler,
+		},
+		fx.Annotated{
+			Name:   "getAllHandler",
+			Target: newGetAllItemsHandler,
+		},
+		fx.Annotated{
+			Name:   "deleteHandler",
+			Target: newDeleteItemHandler,
+		},
```

Comment:
This is all in camel case but usually we use snake case...do you want to update this while you're here?